src/Application.cpp

```diff
@@ -156,8 +156,12 @@ Application::Application(int& argc, char** argv) :
             readOnly = true;
         } else if(arguments().at(i) == "-c" || arguments().at(i) == "--config") {
             // This option has already been handled above
-            // Also since this option was required one more argument(configuration file), it skips one argument
-            i++;
+            // For here, only print the error when no parameter value is given
+            if(++i >= arguments().size())
+            {
+                qWarning() << qPrintable(tr("The -c/--config option requires an argument"));
+                qWarning() << qPrintable(tr("So the -c/--config option is ignored and run application"));
```

Comment:
I'd unify these two messages in a simpler one:
qWarning() << qPrintable(tr("The -c/--config option requires an argument. The option is ignored."));

When it's ignored, it should be obvious that the application will run. Less work for translators :smiley: